app/javascript/src/components/_map.scss

```diff
@@ -21,3 +21,19 @@
   left: calc(((var(--viewport-x) / 2) - var(--x)) * 1px);
   top: calc(((var(--viewport-y) / 2) - var(--y)) * 1px);
 }
+
+.current-map__token-drawer {
+  @apply
+    h-screen
+    bg-gray-500
+  ;
+
+  --zoom-amount: 1.5;
```

Comment:
Sorry, It's not entirely clear to me what this is used for and why its `1.5` can you help me understand?

Reply:
The tokens would usually inherit the `--zoom-amount` variable from the `.current-map`. This would be used to scale them based on the zoom level of the map.

This was my quick way to use the same token element but override the size I would like them to render at when they're in the `.current-map__token-drawer`